.github/ISSUE_TEMPLATE/ask-a-question.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Ask a question
+about: Something is not clear
+title: ""
+labels: ""
+assignees: ""
```

Comment:
I believe labels is an array and assignees a list.

Reply:
Both can be comma-delimited strings; https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms. Ill add appropriate default label, `question`, `bug`, `feature`, for each template.